app/leagues/nfl_hensley.py

```diff
@@ -16,7 +16,7 @@ def build_league(optimize: bool = False) -> League:
     get_score = hensley_cdf_builder(league)
     for g in league.games:
         g.set_score(get_score(g))
-    params = [10135.827483152862, 11418.89391604717]
+    params = [10135.827483152862, 11418.89418889558]
```

Comment:
This change is after adding the week 17 games. I wonder if I'm in a local min or if there's some weirdness with how the optimize function works here